<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>del_classes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 242, 222);</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>511</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Номер</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Буква</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>План</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Здание</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="combo_class">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить класс</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_del_class">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Классы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>231</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../pictures/pngwing.com (6).png</pixmap>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
